### PR TITLE
review: remove 'Disk encryption' section and conditionally hide 'Account'

### PR DESCRIPTION
**Pull request by KKoukiou**
If a device is encrypted is shown as a column in the storage overview.

### CODE REVIEW

**Comment by martinpitt** [comment]
I fixed that error in #351

**Review by M4rtinK — APPROVED** [review]
Looking good as far as I can tell.

File: src/components/review/ReviewConfiguration.jsx [hunk 1]
```diff
@@ -103,17 +103,19 @@ const ReviewConfiguration = ({ idPrefix, setIsFormValid }) => {
...
+                            <ReviewDescriptionListItem
+                              id={`${idPrefix}-target-system-account`}
+                              term={_("Account")}
+                              description={accounts.fullName ? `${accounts.fullName} (${accounts.userName})` : accounts.userName}
+                            />
+                        </ReviewDescriptionList>
+                        <ReviewDescriptionList>
+                            <HostnameRow />
+                        </ReviewDescriptionList>
+                    </>}
```
Comment (jkonecny12) [inline src/components/review/ReviewConfiguration.jsx:118]: Do I understand it correctly that empty field will hide visible parts?

**Review by jkonecny12 — APPROVED** [review]
Looks good as far as I can tell :)

**Comment by jkonecny12** [comment]
Please add commit comment why you are hiding the disc encryption.

File: src/components/review/ReviewConfiguration.jsx (hunk 1 above)
Comment (KKoukiou) [reply src/components/review/ReviewConfiguration.jsx:118]: Ehm you mean <>? This is just wrapper element in react, without reflection on the DOM.

**Comment by KKoukiou** [comment]
> Please add commit comment why you are hiding the disc encryption.

There is reasoning in the commit: If a device is encrypted is shown as a column in the storage overview.
The overview in other words already contains this information. This was leftover before the storage overview was analytic display of the layout.

**Pull request merged**